node/tests/RedisClient.test.ts

```diff
@@ -110,24 +125,23 @@ describe("RedisClient", () => {
     it.each([ProtocolVersion.RESP2, ProtocolVersion.RESP3])(
         "info without parameters",
         async (protocol) => {
-            const client = await RedisClient.createClient(
-                getOptions(port, protocol),
+            client = await RedisClient.createClient(
+                getOptions(cluster.addresses(), protocol),
             );
             const result = await client.info();
             expect(result).toEqual(expect.stringContaining("# Server"));
             expect(result).toEqual(expect.stringContaining("# Replication"));
             expect(result).toEqual(
                 expect.not.stringContaining("# Latencystats"),
             );
-            client.close();
```

Comment:
why?

Reply:
All tests are being run on a single cluster, so I want after every test to flush the cluster and then close the client, so I don't want to close the client before I flushed the cluster using FLUSALL